client-app-sdk/config.yml

```diff
@@ -30,7 +30,7 @@ steps:
     * The configured app should have a URL defining the pcLangTag and pcEnvironment tags
 
     ```
-    ex. https://myapp.com?langTag={pcLangTag}&environment={pcEnvironment}
+    ex. https://myapp.com?langTag={{pcLangTag}}&environment={{pcEnvironment}}
```

Comment:
This ultimately gets rendered in a page by the current dev center, which considers double curlys syntax for templating. These need to be escaped.